src/pages/App/index.tsx

```diff
@@ -38,9 +38,14 @@ function App(): JSX.Element {
         const tokenResult = await user.getIdTokenResult();
         const { claims, token } = tokenResult;
 
+        // TODO: Add ApiConfigStore.setToken(token || '') here so getUserRole() works
+
+        const id = parseInt(claims.user_id, 10);
+        const userRole = await getUserRole(id);
```

Comment:
don't await on both here - make them promises and do a promise.all - i want both awaits to go out together